tests/test_core.py

```diff
@@ -1326,9 +1331,9 @@ def test_polymorph(self):
   def test_complex(self):
     self.do_run_in_out_file_test('tests', 'core', 'test_complex', force_c=True)
 
+  @no_js_backend('no __builtin_fmin support in JSBackend')
```

Comment:
If we only have this one use, this might be overengineered. We `skip` for plenty of other one-off reasons, we probably don't need `@no_whatever` annotations for all of them. (Although a quick skim does suggest `@no_wasm` and `@no_windows` might be useful...)

Reply:
We may not even need this use, running the test now it passes in almost all modes - I suspect we got those builtins with the latest llvm update.

Only one mode fails, -O0 in asm.js (but not wasm), with this:
``
-inf : 3.091515 : 3.091515 : -nan
+inf : 3.091515 : 3.091515 : nan
```
I can look into that.

Reply:
I suspect it's NaN canonicalization in JS - in asm.js without the optimizer, this can happen. We could accept both as valid outputs, or we could pick a value that isn't a nan there (since it's just by chance that `acos` on the silly number `1234.5678` is a nan, I don't think that was intentional).

Reply:
Interesting!  I got the exact same failure when linking with lld.

Reply:
I think I wrote this test? I don't think it's a necessary result and we could (maybe should) change the input.